text/0000-minimal-builders.md

```diff
@@ -0,0 +1,246 @@
+# Minimal Builders
+
+## Summary
+
+Paketo builders are a way of distributing Paketo buildpacks and stacks along
+with the CNB lifecycle. Currently, Paketo distributes the Full, Base and Tiny
+builders, which include different sets of buildpacks and whose stacks each
+have different mixins. While they are a simple way of packaging all the
+necessary components for a Paketo build, they often include more than a user
+actually needs. This RFC proposes that Paketo publish "minimal" versions of its
+three builders. These will include the same build- and run-images as their
+standard counterparts, but will include **no buildpacks nor orders**. This will
+speed up builds by reducing the number of unnecessarily-pulled layers, without
+forcing Paketo users to maintain their own builders if they want to curate the
+buildpacks that are included in a build.
+
```

Comment:
Are there any numbers on how much of a tangible impact this change could make? 

For example, if it reduces the image size by X, and a pipeline is pulling Y times per day on avg, we can estimate a savings of Z by switching them to use a minimal builder like this. Similarly, do we have a comparison of the execution time for a typical pipeline job using the current builders versus a demo of the proposed builders?

I think it would be helpful to gauge the impact in terms of saved bandwidth, time, cost, carbon footprint, or something like that, given that there will be some additional overhead for maintaining and running pipelines to distribute the additional builders.

Reply:
The information you added is excellent. Thanks